### DISCUSSION TITLE
How to change input validated field border colour?

### DISCUSSION THREAD

**Discussion by Smyczek** [mdbootstrap/TW-Elements#1952] [Support from community] [answered]
Hi, 
I'm a newbie and I need help. I cannot change the colours of the valid/invalid input field border in any way. 
I am able to change colour of the notch using 'data-to-class-notch-leading' when field isn't validated but 'data-te-class-notch-leading-valid' doesn't work. What am I doing wrong? This is my code:
<img width="986" alt="Screenshot 2023-09-18 at 21 07 38" src="https://github.com/mdbootstrap/Tailwind-Elements/assets/3749053/e6815d63-7c2b-4f15-8b36-727f8d1487d1">

**Comment by juujisai** [comment, marked as answer]
Hi! Add the `data-te-class-` attribute to the elements that has `data-te-validation-init` data attribute.

```
    <form id="form-1" 
      data-te-validation-init
      data-te-class-notch-leading-valid="border-[#ff0] dark:border-[#ff0] group-data-[te-input-focused]:shadow-[-1px_0_0_#ff0,_0_1px_0_0_#ff0,_0_-1px_0_0_#ff0] group-data-[te-input-focused]:border-[#ff0]"
      >
         ...
    </form>
```
